draft-westerlund-tsvwg-dtls-over-sctp-bis.md

```diff
@@ -616,8 +622,7 @@ During resumption, the maximum message size is renegotiated.
    DTLS/SCTP Mandatory Options, in that case the Association will
    proceed as specified in the previous {{DTLS-init}} section.  If the
    peer replies with an INIT-ACK not containing all DTLS/SCTP
-   Mandatory Options, the Client can decide to keep on working with
-   RFC 6083 fallback, plain data only, or to ABORT the association.
+   Mandatory Options, the Client SHOULD reply with ABORT.
```

Comment:
This is hard to realise... At least with SCTP implementations using the socket API specified in [RFC 6458](https://datatracker.ietf.org/doc/html/rfc6458). The normal procedure is that the upper layer enables some options, sets some parameters, then the association is set up and features are negotiated. After the association has been established, the end-points can check and abort the association.

The constraints are coming from the upper layer, so the upper layer should check it. In this case, the upper layer is the DTLS implementation.

Reply:
Sounds like something that can be resolved with an reformulation. I guess the issue here is depending on how one interpret the meaning of "the client". If that is the SCTP layer only, no I can see the validity of your comment under some API, but I think we are talking about the DTLS + SCTP stack combination that needs to do the right thing here.

Reply:
> Sounds like something that can be resolved with an reformulation. I guess the issue here is depending on how one interpret the meaning of "the client". If that is the SCTP layer only, no I can see the validity of your comment under some API, but I think we are talking about the DTLS + SCTP stack combination that needs to do the right thing here.

Yes.

Reply:
I have just committed some changes to these two paragraphs. @tuexen can you please check if this is better or possible suggest rewording?

Reply:
Will do after we finished the handshake discussion.